src/data/cech.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
You should rename this file from `cech.json` to `CECHProjects.json` so that the file name is more descriptive of what it holds